content/blog/2021/04/2021-04-12-jenkins-operator-sub-project.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Jenkins Operator becoming sub-project!"
+tags:
+- kubernetes
+- operator
+- contributing
+authors:
+- antoniaklja
+- sylwiabrant
+sig: docs
+opengraph:
+  image: /images/post-images/2021-04-jenkins-operator/tshirt-logo.jpg
+---
+
+We are happy to announce that Jenkins Operator officially became Jenkins sub-project.
+
+== What does it mean for the project?
+
+Becoming an official part of the Jenkins project was a major step towards better alignment with the overall Jenkins’ roadmap and more opportunities to increase adoption of the Jenkins Operator project.
+
+Finally, with a dedicated team of people actively contributing to the project we can engage with the wider community and participate in some of the Cloud-Native SIG meetings.
+*This opens a room for everyone to voice their opinions or start supporting the project.*
+
+We truly believe that this partnership will result in significant improvements to Jenkins Operator, as well as Jenkins ecosystem itself.
+
+== Bridging the gap between Jenkins and Kubernetes
+
+Running Jenkins in a cloud-native environment like Kubernetes is not a trivial task.
+With Jenkins Operator project we want to enable the community to take full advantage of Kubernetes and public cloud capabilities by:
+
+* native integration with public cloud services in areas of observability, storage and cloud security
+* Kubernetes autoscaling and self-healing mechanism
+* secure access to Jenkins instance
+* declarative configuration using Kubernetes Custom Resources
+* full lifecycle management, eventually transforming it to autopilot
+
+== Take part in the journey toward global trends
+
+*Start contributing and play an important role in creating the automated Jenkins experience! Don’t hesitate - engage in the community.*
+Join us in our work of creating the functionalities you see beneficial.
+You are welcome to create issues and pull requests. We are actively resolving community issues and providing answers on a dedicated Slack channel.
+
+== Lead the way we are heading by providing us feedback
+
+To celebrate the new possibilities that are being opened for our project, we would like to invite you to take part in a short survey that will help put us on the right track.
+If you have been using Jenkins Operator or running Jenkins in any other environment, please take a moment to fill out our quick survey.
+*We will choose at least three of the most informative answers and send you an awesome Jenkins Kubernetes Operator T-shirt with our cute gopher butler.*
+The survey can be found link:https://docs.google.com/forms/d/1doIkgnm3_WbjtlwWSU4sOoiI7QoneHlYIjXEJOVMrfQ/edit?usp=sharing[here]. Remember, the most sincere answers are the best.
+
+image:/images/post-images/2021-04-jenkins-operator/tshirt-logo.jpg[T-shirt, width=400px]
+
+And if you still haven’t used Jenkins Kubernetes Operator, you’re more than welcome to do so. Give it a shot and discover a new kind of simplicity of running Jenkins.
```

Comment:
Maybe it makes sense to link or even embed the presentation from @tomaszsek : https://www.youtube.com/watch?v=gpKedb038S0 . It could be a good quick start overview if it is still compatible with the recent versions

Reply:
we are planning a brand new presentation soon :)